app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -358,6 +365,9 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             updateToolbarContentDescription(it.searchEngineSource)
             toolbarView.update(it)
             awesomeBarView.update(it)
+            if (requireContext().settings().showUnifiedSearchFeature) {
+                addQrButton(it)
+            }
             addVoiceSearchButton(it)
```

Comment:
```suggestion
            addVoiceSearchButton(it)
            
            if (requireContext().settings().showUnifiedSearchFeature) {
                addQrButton(it)
            }

```

Reply:
We need to add the QR button before the voice button.  This is so we keep the order correct in UX.